hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/TestViewDistributedFileSystem.java

```diff
@@ -67,4 +70,23 @@ public void testOpenWithPathHandle() throws Exception {
       }
     }
   }
+
+  @Override
+  public void testEmptyDelegationToken() throws IOException {
+    Configuration conf = getTestConfiguration();
+    MiniDFSCluster cluster = null;
+    try {
+      cluster = new MiniDFSCluster.Builder(conf).numDataNodes(1).build();
```

Comment:
nit:
you aren't writing any data, the numDatanodes can be set to 0.

Reply:
Done. Thanks